Dockerfile

```diff
@@ -161,19 +176,25 @@ COPY scripts /scripts
 RUN curl -sL https://github.com/jwilder/dockerize/releases/download/v0.6.1/dockerize-linux-amd64-v0.6.1.tar.gz | tar zxf - -C /bin/
 
 # Update access of required directories to allow not running in root for openshift
-RUN chmod -R g+rw ${CATALINA_HOME}/logs  && \
+RUN mkdir -p ${CATALINA_HOME}/work/Catalina/localhost/prweb && \
+    chmod -R g+rw ${CATALINA_HOME}/logs  && \
     chmod -R g+rw ${CATALINA_HOME}/lib  && \
     chmod -R g+rw ${CATALINA_HOME}/work  && \
     chmod -R g+rw ${CATALINA_HOME}/conf  && \
     chmod -R g+rw ${CATALINA_HOME}/bin  && \
     chmod -R g+rw ${CATALINA_HOME}/webapps && \
-    chmod -R g+x /scripts && \
-    chown -R pegauser /scripts && \
     chmod g+r ${CATALINA_HOME}/conf/web.xml && \
     chown -R pegauser ${CATALINA_HOME}  && \
+    chmod -R o+rx ${CATALINA_HOME} && \
```

Comment:
Perhaps to simplify things a bit here, how about we chmod -R 775 $(CATALINA_HOME)?  This would eliminate lines180-186 and would be a lot easier to figure out what is going on.

Reply:
We can probably do this, and then grant 777 for logs/temp dir